docker-compose/obam-with-obkm/scripts/deploy.sh

```diff
@@ -30,25 +30,33 @@ function echoBold () {
     ${ECHO} -e $'\e[1m'"${1}"$'\e[0m'
 }
 
-read -p "Enter Your Open-Banking Specification (UK[1]/Berlin[2]): " SPEC
+read -p "Enter Your Open-Banking Specification (UK[1]/Berlin[2]/AU[3]): " SPEC
 ${ECHO}
 
 if [[ ${SPEC} =~ "1" ]]; then
-    if ! ${SED} -i.bak -e 's|/berlin|/uk|' ../docker-compose.yml; then
+    if ! ${SED} -i.bak -e 's|/berlin|/uk|;s|/au|/uk|' ../docker-compose.yml; then
     echoBold "Could not configure to use the WSO2 Docker image available at DockerHub"
```

Comment:
@imesh94 just to clarify, for OB no Docker images are available at the public WSO2 organization in DockerHub [1].

Instead, Docker images available at WSO2 Private Docker registry [2] are used. For detailed information, please refer [3].

Hence, shall we please update the error message to refer to the WSO2 Private Docker Registry instead of the DockerHub? For example,

```
Could not configure to use the WSO2 Docker image available at WSO2 Private Docker registry
```

[1]: [WSO2 organization at DockerHub](https://hub.docker.com/u/wso2/)
[2]: [WSO2 Private Docker registry](https://docker.wso2.com/)
[3]: [WSO2 Products Dockerized: An Introduction](https://medium.com/@chirangaalwis/wso2-products-dockerized-an-introduction-5045a197befa)

Reply:
done